lib/commands/project.js

```diff
@@ -352,6 +344,23 @@ project.create = function (projectName, servoSize, runtime, cb) {
         fn();
       });
     },
+    function findOrganizationMembers(fn) {
+      userController.getOrganizations(userConfig.data.userId, function(err, results) {
```

Comment:
`mod-standard` is reporting `Unexpected space before function parentheses` when I add the space there.